src/tic_tac_toe/utils.clj

```diff
@@ -1,8 +1,47 @@
 (ns tic-tac-toe.utils
```

Comment:
I try to avoid namespaces with names like `utils` or `helpers` because it tends to become a collection of disparate things.

I feel like there is at least one namespace that can be extracted here, maybe more.

Reply:
Split this to three places:
[save and exit](https://github.com/saralein/tic-tac-toe_clojure/pull/10/files#diff-d484ebc9d5c821d74215acabe6b6300bR1)
[menu selector](https://github.com/saralein/tic-tac-toe_clojure/pull/10/files#diff-780f5553485d26d4062ac39d7fd46a79R1)
[back into game](https://github.com/saralein/tic-tac-toe_clojure/pull/10/files#diff-1e397d0c219d54561be23ddb30fd0516R24) - noticed this bit of logic was also used in the referee; thought it made sense to move it to game as retry-move